README.md

```diff
@@ -190,6 +190,7 @@ Gauges:
 * bbb_meetings_listeners - Total number of listeners in all BigBlueButton meetings
 * bbb_meetings_voice_participants - Total number of voice participants in all BigBlueButton meetings
 * bbb_meetings_video_participants - Total number of video participants in all BigBlueButton meetings
+* bbb_meetings_participant_clients(labels: client) - Total number of participants in all BigBlueButton meetings by client (html5|dial-in|flash)
```

Comment:
I would maybe change this to: `bbb_meetings_participant_clients(type=<client>)`.